examples/07-periodic_hills/periodic_hills.prm

```diff
@@ -98,7 +109,7 @@ end
 subsection non-linear solver
   set tolerance                   = 1e-5
   set max iterations              = 10
-  set verbosity                   = verbose
+  set verbosity                   = quiet
```

Comment:
Keep the verbosity for the examples. Those are good things to take a habit of looking at!

Reply:
Actually, I have commited the .prm file but I didn't want to, so it's not the final .prm for the example, it was only for testing.
But I will keep the verbosity for the final .prm, I promise :)